client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx

```diff
@@ -56,11 +56,34 @@ const ContactQuestioningClinical: React.FC<Props> = (props: Props): JSX.Element
     const isolationEndDate = addDays(new Date(interactedContact.contactDate), daysToIsolate);
     const formattedIsolationEndDate = format(new Date(isolationEndDate), 'dd/MM/yyyy');
 
+    const formatContactToValidate = () => {
+        let newConatct = formValues;
+        newConatct.firstName = interactedContact.firstName;
+        newConatct.lastName = interactedContact.lastName;
+        newConatct.contactType = interactedContact.contactType;
+
+        return newConatct;
```

Comment:
Could use deconstruction to create new object based on form values

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/998/commits/5a571cf82d9b4306dd23f29a6445e3ab6ab60a23)